### ISSUE TITLE
Print paths to tests that should be rerun 

### ISSUE THREAD

**Issue by henningphan** [tarpas/pytest-testmon#93]
Enhancement,

In my personal setup, I have a different system that runs the tests.

It would be great if testmon had the option to print the fullpath of the affected tests.

**Comment by tarpas** [comment]
related to #62

**Comment by henningphan** [comment]
I found out that in my particular scenario, you could just print all selected items in def pytest_modify..().
and then move all selected items to deselected.
Tell me if my solution is of interest.

I find your #62 to be more ambitious than my needs

**Comment by tarpas** [comment]
unfortunatelly I plan difficult refactoring of pytest_testmon.py, so I'm not very keen to add functionality there before. Wouldnt a better UI be separate command which prints just the test names, so that it could be used as input to other commands directly? (without parsing out pytest headers)?

**Comment by henningphan** [comment]
I agree a separate command which just print the test names is a more optimal solution. Could you point out how one could accomplish that?

**Comment by tarpas** [comment]
Not easily, sorry. Your strategy of printing from pytest_modify_collection is the right one. I will just not be able to merge it. After refactor it should be very easy to create a separate plugin class which does only what you request.

**Comment by tarpas** [comment]
@henningphan do you still need this functionality? I can start working on it if still needed.

**Comment by henningphan** [comment]
Hello, Im not longer on the project and therefore dont need this feature.
